docs/sources/operators-guide/configuring/configuring-out-of-order-samples-ingestion.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configuring out-of-order samples ingestion"
+menuTitle: "Configuring out-of-order samples ingestion"
+description: "Learn how to configure Grafana Mimir to handle out-of-order samples ingestion."
+weight: 120
+---
+
+# Configuring out-of-order samples ingestion
+
+Grafana Mimir and the Prometheus TSDB understand out-of-order as follows.
+
+The moment that a new series sample arrives, you need to determine if the series already exists, and whether or not the sample is too old:
+
+- If the series exists, it must be newer than the latest sample that is stored.
+  Otherwise, it is considered out-of-order and will be dropped by the ingesters.
+- If the series does not exist, then it has to be newer than TSDB's head-block maximum time. If it isn’t newer, then it is also considered out-of-order and will be dropped by the ingesters.
```

Comment:
> then it (incoming sample) has to be newer than TSDB's head-block maximum time.

This isn't quite true, right? Incoming sample can be within "bounds", which goes 1h back from Heads' maxTime (when using 2h block range).

Reply:
Sorry this is what I meant but I wrote it wrong.